composer.json

```diff
@@ -12,24 +12,25 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "~2.1",
-        "symfony/process": "~2.1",
-        "symfony/finder": "~2.1",
+        "php": "^5.5|~7.0",
+        "symfony/framework-bundle": "~2.8|~3.0",
```

Comment:
Yeah but we will need to fork the versions... Maybe this should be tagged as a new major, and the old major can still be mantained?